apps/users/models/committee.py

```diff
@@ -6,7 +6,7 @@
 
 
 def validate_aspect_ratio(image):
-    ratio = 16 / 9
+    ratio = 1 / 1
```

Comment:
what happens to old committee photos currently in the db? do all of them throw ValidationErrors(s) or is it only when the db gets changed?

Reply:
only when you want to change it will it matter